webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
```

Comment:
Your webpack produce the warming below;
![image](https://user-images.githubusercontent.com/31897434/103165690-2959cd00-481b-11eb-948f-3cd4fa3aa537.png)

Let's try to compress the codebase;

Try to add the block of code below to the `module.exports` object.

```
  devServer: {
    contentBase: path.join(__dirname, 'dist'),
    compress: true,
  },
```

Reply:
Script commands are working at my end. I sent a message to you on slack. If someone tells me what the error is, I will try to fix it. I cannot solve errors that I'm not facing.